lib/ui/painting.dart

```diff
@@ -2079,6 +2079,16 @@ class Path extends NativeFieldWrapperClass2 {
   }
   Path _transform(Float64List matrix4) native 'Path_transform';
 
+  /// Cuts the path into a subset of itself leaving only the region between
+  /// `startT` and `stopT` fraction values. Set `isComplement` to true to
+  /// invert the result by cutting out the region from `startT` to `stopT`.
+  ///
+  /// Returns true when the cut is succesfully made, false if not.
+  bool trim(double startT, double stopT, bool isComplement) {
```

Comment:
Can isComplement just be isInverted to match Skia terminology?

Reply:
Yes, that is definitely more clear.